test/module/irohad/ametsuchi/rocksdb_common_test.cpp

```diff
@@ -61,6 +80,209 @@ class RocksDBTest : public ::testing::Test {
   std::string const value5_ = "vaLUe5";
 };
 
+TEST_F(RocksDBTest, DatabaseCacheTest) {
+  iroha::ametsuchi::DatabaseCache<std::string> dbc;
+  dbc.addCacheblePath("wSc");
+  dbc.addCacheblePath("wScq");
+  dbc.addCacheblePath("bps");
+  dbc.addCacheblePath("bps");
+  dbc.addCacheblePath("bpsQ");
+  dbc.addCacheblePath("bpsQ0");
+  dbc.addCacheblePath("bpm");
+
+  dbc.addCacheblePath("12");
+  dbc.addCacheblePath("1");
+
+  std::string src[] = {"bps1", "1jg", "0pp", "2"};
+
+  {
+    size_t counter = 0ull;
+    for (auto &s : src)
+      if (dbc.isCacheable(s)) {
+        dbc.set(s, s + "_value");
+        ++counter;
+      }
+    ASSERT_EQ(counter, 2ull);
+  }
+
+  auto check = [](auto const &str1, auto const &str2) {
+    ASSERT_EQ(str1, str2);
+  };
```

Comment:
Without that lambda you would have less LOCs :)

Reply:
It's because ```ASSERT_EQ``` makes ```return```.